services/movie_session.py

```diff
@@ -42,3 +42,14 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
Consider using `List[Dict]` from the `typing` module for the return type annotation to indicate that the function returns a list of dictionaries.